rts/Sim/Units/CommandAI/CommandAI.cpp

```diff
@@ -701,9 +700,10 @@ bool CCommandAI::AllowedCommand(const Command& c, bool fromSynced)
 
 				if (attackee == nullptr)
 					return false;
-				if (!attackee->pos.IsInBounds())
-					return false;
 			} else {
+				if (c.GetNumParams() > 3 && c.GetParam(3) == 0 && !IsCommandInMap(c))
```

Comment:
You can probably give 3-param orders (without the 4th radius arg) via Lua which I think would also be a valid ground attack but not caught here. If so then
```suggestion
				const bool isGroundTargeted = (c.GetNumParams()  > 3 && c.GetParam(3) == 0)
				                           ||  c.GetNumParams() == 3;
				if (isGroundTargeted && !IsCommandInMap(c))
```

Reply:
Done